src/types.tsx

```diff
@@ -66,6 +66,8 @@ export interface FormikComputedProps<Values> {
  * Formik state helpers
  */
 export interface FormikActions<Values> {
+  /** Get all registered fields. */
+  getAllRegisteredFields(): Object;
```

Comment:
I'd use `{ [fieldName: string]: React.Component<any> }` as a type. Also it doesn't quite fit to `FormikActions<Values>` interface. What about moving it to `FormikComputedProps<Values>` and changing it to a getter?

Reply:
Hmm... maybe we could secure it even more. What about changing it to an array of field names? We only need those and it would always be read-only 🔒  😄